task/grype/0.1/grype.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       description: The Arguments to be passed to Grype command.
       type: array
     - name: GRYPE_IMAGE
-      default: docker.io/anchore/grype@sha256:40c41e43eff0eb979375ea705f0cf5f6f698941099f63c890c3eb90b6ef72db5 # v0.47.0
+      default: docker.io/anchore/grype@sha256:ce8398ad2028c2181b4d626e1902cf98e1adde22ff6dca9530deada21b1a9b97  # latest
```

Comment:
generally we should avoid taking image from the latest tag as we won't get to know the version of the binary in future. Can you please take any image digest from a tagged version?